PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/secure/IdeaTrustStrategy.kt

```diff
@@ -57,10 +59,21 @@ object IdeaTrustStrategy : TrustStrategy, ILogger {
                 log().warn("First check untrusted X509 certificates chain $chain for authentication type $authType", err)
             }
 
-            //as for case3, sysmgr always throws ValidatorException exception then customMgr popups ca accept or not dialog
-            //if user accepts,return true
-            //if user rejects,ValidatorException will rethrow then return false
-            //otherwise return false when exp happens during mgr checking cert
+            // as for case3,if user already chooses bypass cert check
+            if (HttpObservable.isSSLCertificateValidationDisabled()) {
```

Comment:
Why not put the code into the `catch` clause above?

Reply:
it will pop up the dialog first,but after check,it will directly add the cert